widget/embedded/src/pages/AddCustomTokenPage.tsx

```diff
@@ -177,10 +177,10 @@ export function AddCustomTokenPage() {
             }>
             <Divider size={40} />
             <Divider size={10} />
-
+            {/* eslint-disable-next-line jsx-id-attribute-enforcement/missing-ids */}
```

Comment:
Are you sure we need this comment? The ID is already being passed to the button.